src/accelerate/commands/config/cluster.py

```diff
@@ -298,6 +298,18 @@ def get_cluster_input():
                         "When `zero3_init_flag` is set, it requires Transformers to be installed. "
                         "Please run `pip3 install transformers`."
                     )
+            use_moe = _ask_field(
+                "Do you want to enable MoE? [yes/NO]: ",
```

Comment:
Maybe spell out MoE for those who don't know the acronym?

Reply:
Done